### DISCUSSION TITLE
List the Categories (or Tags) in the same way as the Archives

### DISCUSSION THREAD

**Discussion by a0damasco** [CaiJimmy/hugo-theme-stack#559] [Q&A]
Hi!
![image](https://user-images.githubusercontent.com/100387433/158074992-0e165e7e-6b8d-4505-b991-5c521ec7feaa.png)
The Categories and Tags widget are displayed in a "cloud" format, how can I list them in the same way as the Archives?

**Comment by CaiJimmy** [comment]
> How can I list them in the same way as the Archives?

It's not possible currently unless you modify the source code 😥.
